### PR TITLE
Added compass primary defaults (mttr-4.0)

### PR DESCRIPTION
**Pull request by Aksellence**
Added compass primary default for setting the CAN compass as primary and I2C compass as a secondary fallback heading reference sensor.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm [hunk 1]
```diff
@@ -13,3 +13,7 @@ SERIAL5_PROTOCOL 1
 # lower heater gains
 BRD_IMUHEAT_I 0.07
 BRD_IMUHEAT_P 50
+
+# compass primary defaults
+COMPASS_PRIO1_ID 73475
```
Comment (zalice) [inline libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm:18]: This should be
```
COMPASS_PRIO1_ID COMPASS_DEV_ID2
COMPASS_PRIO2_ID COMPASS_DEV_ID
```

**Review by zalice — APPROVED** [review]

**Review by zalice — CHANGES_REQUESTED** [review]
Please see requested param values

File: libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm (hunk 1 above)
Comment (Aksellence) [reply libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm:18]: I don't think so. As per me, it should be the value associated with the particular compass_dev_id that we want at a certain priority and not the string mentioning the 'compass_dev_id<ID number>'

Comment (tridge) [reply libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm:18]: using the string won't work, as the parser doesn't know how to decode parameter names in the right hand column

File: libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm [hunk 2]
```diff
@@ -13,3 +13,7 @@ SERIAL5_PROTOCOL 1
...
+COMPASS_PRIO2_ID 592905
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm]: I'd also suggest setting COMPASS_PRIO3_ID to 0, to ensure it doesn't have a conflict

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm (hunk 1 above)
Comment (zalice) [reply libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm:18]: Thank you for the clarification, @tridge !

**Review by zalice — APPROVED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm (hunk 2 above)
Comment (Aksellence) [reply libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm]: Done. Thanks!

**Review by zalice — APPROVED** [review]

**Pull request merged**